src/css/wizard-navigation-bar.scss

```diff
@@ -291,119 +306,89 @@ aw-wizard-navigation-bar {
       }
 
       a:hover .label {
-        color: darken($wz-color-current, 20%);
+        color: $wz-label-color-hover;
       }
     }
   }
 }
 
-aw-wizard-navigation-bar.horizontal {
+@if 'horizontal' is in $wz-layouts {
+  aw-wizard-navigation-bar.horizontal {
 
-  ul.steps-indicator {
-    display: flex;
-    flex-direction: row;
-    justify-content: center;
-
-    margin: 0;
-    width: 100%;
-    list-style: none;
-
-    @mixin steps($number-of-components) {
-      &:before {
-        left: 100% / $number-of-components / 2;
-        right: 100% / $number-of-components / 2;
-      }
+    ul.steps-indicator {
+      display: flex;
+      flex-direction: row;
+      justify-content: center;
 
-      li {
-        width: 100% / $number-of-components;
-      }
-    }
-
-    &.steps-2 {
-      @include steps(2);
-    }
-
-    &.steps-3 {
-      @include steps(3);
-    }
-
-    &.steps-4 {
-      @include steps(4);
-    }
-
-    &.steps-5 {
-      @include steps(5);
-    }
-
-    &.steps-6 {
-      @include steps(6);
-    }
-
-    &.steps-7 {
-      @include steps(7);
-    }
-
-    &.steps-8 {
-      @include steps(8);
-    }
-
-    &.steps-9 {
-      @include steps(9);
-    }
-
-    &.steps-10 {
-      @include steps(10);
-    }
-
-    li {
       margin: 0;
-      padding: 0;
-      text-align: center;
+      width: 100%;
+      list-style: none;
+
+      @for $number-of-components from 2 through 10 {
```

Comment:
Does it make sense to use `1` as the minimum count? I don't see a reason why there shouldn't be a usecase where a wizard contains only a single step...

Reply:
Wizard with a single step already looks perfectly without additional styles enabled by this `@for`. I have just created a demo page in angular-archwizard-demo project: https://github.com/madoar/angular-archwizard-demo/pull/49/commits/e205af439dfcb0771af9f4b33a354fb8aa14c409

Reply:
:+1: